asf_heat_pump_suitability/pipeline/suitability/calculate_suitability.py

```diff
@@ -256,46 +56,47 @@ def compute_df_total_score_per_epc(
+        grid_installation_threshold: minimum percentage of properties in LSOA that could have a heat pump installed with current grid capacity
 
     Returns:
         pl.DataFrame: suitability score for specified tech type
     """
     df = df.with_columns(
         pl.when(pl.col("ruc_two_fold") == "Urban")
-        .then(high_heat_demand_scores.get(tech_type))
+        .then(scoring.high_heat_demand_scores.get(tech_type))
         .alias("heat_demand_score"),
-        pl.when(pl.col("OFF GAS"))
-        .then(offgas_scores.get(tech_type))
+        pl.when(pl.col("off_gas"))
+        .then(scoring.offgas_scores.get(tech_type))
         .alias("off_gas_score"),
         pl.when(pl.col("households_per_km2") > density_threshold)
-        .then(property_density_scores.get(tech_type))
+        .then(scoring.property_density_scores.get(tech_type))
         .alias("property_density_score"),
         pl.when(pl.col("garden_area_m2") > garden_threshold)
-        .then(garden_size_scores.get(tech_type))
+        .then(scoring.garden_size_scores.get(tech_type))
         .alias("garden_size_score"),
         pl.when(pl.col("garden_area_m2") > external_space_threshold)
-        .then(external_space_scores.get(tech_type))
+        .then(scoring.external_space_scores.get(tech_type))
         .alias("external_space_score"),
         pl.when(~pl.col("listed_building"))
-        .then(site_regs_scores.get(tech_type))
+        .then(scoring.site_regs_scores.get(tech_type))
         .alias("not_listed_score"),
-        pl.when(~pl.col("in_conservation_area"))
-        .then(site_regs_scores.get(tech_type))
-        .alias("not_in_cons_area_score"),
+        pl.when(~pl.col("in_protected_area"))
+        .then(scoring.site_regs_scores.get(tech_type))
+        .alias("not_in_protected_area_score"),
         pl.when(pl.col("property_type") != "Flat, maisonette or apartment")
-        .then(not_flat_scores.get(tech_type))
-        .otherwise(multiple_props_scores.get(tech_type))
+        .then(scoring.not_flat_scores.get(tech_type))
+        .otherwise(scoring.multiple_props_scores.get(tech_type))
         .alias("property_type_score"),
         pl.when(pl.col("CURRENT_ENERGY_RATING").is_in(["A", "B", "C"]))
-        .then(epc_threshold_scores.get(tech_type))
+        .then(scoring.epc_threshold_scores.get(tech_type))
         .alias("epc_rating_score"),
         pl.when(pl.col("has_anchor_property"))
-        .then(anchor_properties_scores.get(tech_type))
+        .then(scoring.anchor_properties_scores.get(tech_type))
         .alias("anchor_properties_score"),
         pl.when(
-            pl.col("heatpump_installation_percentage") > grid_installation_threshold
+            pl.col("heatpump_installation_percentage") >= grid_installation_threshold
         )
-        .then(epc_threshold_scores.get(tech_type))
+        .then(scoring.epc_threshold_scores.get(tech_type))
         .alias("grid_capacity_score"),
```

Comment:
Thanks @lizgzil ! Great spot on the `epc_threshold_scores`. Weird that we didn't use the `grid_capacity_scores` dict before because it did already exist so not sure what happened! I've updated it now.

I've also updated the scoring calculation based on your suggestion which I think is a good idea because it gives more nuance in the final scores. So now, if the `heatpump_installation_percentage` is not null, then the score will be multiplied by the %. For this reason, we no longer require the `grid_installation_threshold` arg so I have removed it.